changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - Montana interest exemptions.
```

Let's use the name in the law
```suggestion
    - Montana senior interest income exclusion.
```